docs/core/versions/selection.md

```diff
@@ -29,83 +29,151 @@ SDK commands include `dotnet new` and `dotnet run`. The .NET CLI must choose an
 
-- ✔️ 3.0 is specified. 3.0.3 is the highest patch version installed. 3.0.3 is used.
-- ❌ 3.0 is specified. No 3.0.* versions are installed. 2.1.1 is the highest runtime installed. An error message is displayed.
-- ✔️ 3.0 is specified. No 3.0.* versions are installed. 3.1.0 is the highest runtime version installed. 3.1.0 is used.
-- ❌ 2.0 is specified. No 2.x versions are installed. 3.0.0 is the highest runtime installed. An error message is displayed.
+- ✔️ 5.0 is specified. 5.0.3 is the highest patch version installed. 5.0.3 is used.
+- ❌ 5.0 is specified. No 5.0.* versions are installed. 3.1.1 is the highest runtime installed. An error message is displayed.
+- ✔️ 5.0 is specified. No 5.0.* versions are installed. 5.1.0 is the highest runtime version installed. 5.1.0 is used.
+- ❌ 3.0 is specified. No 3.x versions are installed. 5.0.0 is the highest runtime installed. An error message is displayed.
 
 Minor version roll-forward has one side-effect that may affect end users. Consider the following scenario:
 
-1. The application specifies that 3.0 is required.
-2. When run, version 3.0.* is not installed, however, 3.1.0 is. Version 3.1.0 will be used.
-3. Later, the user installs 3.0.3 and runs the application again, 3.0.3 will now be used.
+01. The application specifies that 5.0 is required.
+02. When run, version 5.0.* isn't installed, however, 5.1.0 is. Version 3.1.0 will be used.
+03. Later, the user installs 5.0.3 and runs the application again, 5.0.3 will now be used.
+
+It's possible that 5.0.3 and 5.1.0 behave differently, particularly for scenarios like serializing binary data.
+
+### Control roll-forward behavior
+
+The roll-forward behavior for an application, can be configured in four different ways:
+
+01. Project-level setting by setting the `<RollForward>` property:
+
+    ```xml
+    <PropertyGroup>
+      <RollForward>LatestMinor</RollForward>
+    </PropertyGroup>
+    ```
+
+02. The `*.runtimeconfig.json` file.
+
+    This file is produced when you compile your application. If the `<RollForward>` property was set in the project, it's reproduced in the `*.runtimeconfig.json` file as the `rollForward` setting. Users can edit this file to change the behavior of your application.
+
+    ```json
+    {
+      "runtimeOptions": {
+        "tfm": "net5.0",
+        "rollForward": "LatestMinor",
+        "framework": {
+          "name": "Microsoft.NETCore.App",
+          "version": "5.0.0"
+        }
+      }
+    }
+    ```
+
+03. The `dotnet` command's `--roll-forward <value>` property.
+
+    When you run an application, you can control the roll-forward behavior through the command line:
+
+    ```dotnet
+    dotnet run --roll-forward LatestMinor
+    dotnet myapp.dll --roll-forward LatestMinor
+    myapp.exe --roll-forward LatestMinor
+    ```
+
+04. The `DOTNET_ROLL_FORWARD` environment setting.
+
+### Precedence
+
+Roll forward behavior is considered in the following order:
```

Comment:
This is confusing (at least to me). Reading this I would assume the `runtimeconfig.json` is the higher priority and thus overrides all of the other options, but it's actually the other way round. I think there should be an explanation of what the order in this list means.

Reply:
OK, I added some descriptions on each of the items.